contracts/src/utils/ProxyLib.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.8;
+
+import {ERC1967Proxy} from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import {Clones} from "@openzeppelin/contracts/proxy/Clones.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+
+library ProxyLib {
```

Comment:
Note, that this library is created to replace [aragon/osx Proxy.sol](https://github.com/aragon/osx/blob/e7ba46026db96931d3e4a585e8f30c585906e1fc/packages/contracts/src/utils/Proxy.sol#L13). Additionally, this provides a method to cloning and directly initializing a minimal proxy contract.